### DISCUSSION TITLE
Postgis - pre-support?

### DISCUSSION THREAD

**Discussion by magick93** [geldata/gel#3727] [General]
Hello

I saw there was already a question asking about whether you'll support postgis.

I'm also very interested in this. My question is, as EdgeDb builds on Postgres, is it possible to install postgis now? Will that break anything in EdgeDb?

Will any of the functions work within EdgeDb?

Can EdgeDb be accesses like traditional postgres?

**Comment by 1st1** [comment]
> I saw there was already a question asking about whether you'll support postgis.

Yes, we want to add support for it. Probably in 3.0 (will be released towards the EOY).

> I'm also very interested in this. My question is, as EdgeDb builds on Postgres, is it possible to install postgis now? Will that break anything in EdgeDb?

We don't expose our Postgres directly, but even if you get to it, just installing postgis won't do much. We still have to properly map Postgis' types & functions to EdgeQL equivalents. We'll be working on this very soon. Stay tuned!
